packages/std/src/encoding.rs

```diff
@@ -48,6 +48,37 @@ impl From<&[u8]> for Binary {
     }
 }
 
+// Macro needed until https://rust-lang.github.io/rfcs/2000-const-generics.html is stable.
```

Comment:
Nicer than cut-and-paste.

Yeah, I came across the const generics talks a number of times in similar contextes